datatypes/src/collections/multi_point_collection.rs

```diff
@@ -258,6 +259,134 @@ impl MultiPointCollection {
                 .clone(),
         ]
     }
+
+    /// Helper function to create json values from the collection's multi points
+    fn coordinates_to_geo_json_geometries(&self) -> impl Iterator<Item = geojson::Geometry> + '_ {
+        let geometry_column: &ListArray = downcast_array(
+            &self
+                .data
+                .column_by_name(Self::FEATURE_COLUMN_NAME)
+                .expect("Column must exist, since it is in the metadata"),
+        );
+
+        (0..self.len()).map(move |i| {
```

Comment:
is there a way to have an iterator over the entries instead of the 0..len range?

Reply:
d387147